CONTRIBUTING.md

```diff
@@ -76,6 +76,22 @@ It is also a good idea to run jest in watch mode while developing on a single pa
 $ ./node_modules/.bin/jest ./packages/<package-name>/tests --watch
 ```
 
+## Run e2e Experience With Smoke Tests
+
+WebdriverIO maintains a smoke test suites that allows to represent the full e2e experience of a user running the wdio testrunner. It is setup in a way so it doesn't require an actual browser driver since all requests are mocked using the `@wdio/webdriver-mock-service`. This offers you an opportunity to run a wdio test suite without setting up a browser driver and a test page. You can run all smoke tests via:
```

Comment:
`WebdriverIO maintains a smoke test suites` if multiple I suggest using `a set of smoke test suites` :)